TravelPoints-BackEnd/src/main/java/com/example/travelpointsbackend/controller/UserController.java

```diff
@@ -0,0 +1,24 @@
+package com.example.travelpointsbackend.controller;
+
+import com.example.travelpointsbackend.model.User;
+import com.example.travelpointsbackend.service.UserService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Optional;
+
+@CrossOrigin(origins = "http://localhost:4200")
+@RestController
+@RequestMapping("/user")
+@RequiredArgsConstructor
+@Slf4j
+public class UserController {
+    private final UserService userService;
+
+    @GetMapping("/getById/{id}")
+    public ResponseEntity<Optional<User>> getUserById(@PathVariable Long id){
+        return ResponseEntity.ok(userService.findById(id));
```

Comment:
M: does this return all the fields from the user, including password?

Reply:
It's just a test controller for security, to see if it blocks what's not in permit all. I don't utilize this controller for sending data to the frontend.
But yes, return all fields including password.